datastore/tests/unit/test_transaction.py

```diff
@@ -212,6 +218,95 @@ class Foo(Exception):
         self.assertIsNone(xact.id)
         self.assertEqual(ds_api.begin_transaction.call_count, 1)
 
+    def test_default(self):
+        project = 'PROJECT'
+        id_ = 850302
+        ds_api = _make_datastore_api(xact=id_)
+        client = _Client(project, datastore_api=ds_api)
+        xact = self._make_one(client)
+        self.assertTrue(isinstance(xact.options, self._get_options_class()))
+
+    def test_read_only(self):
+        project = 'PROJECT'
+        id_ = 850302
+        ds_api = _make_datastore_api(xact=id_)
+        client = _Client(project, datastore_api=ds_api)
+        read_only = self._get_options_class().ReadOnly()
+        options = self._make_options(read_only=read_only)
+        xact = self._make_one(client, read_only=True)
+        xact.begin()
+        begin = ds_api.begin_transaction
+        begin.assert_called_once()
+        self.assertEqual(begin.call_count, 1)
+
+    def test_read_write(self):
```

Comment:
Delete this test? It doesn't use `options` and isn't supported by the constructor.